tests/test_optics.py

```diff
@@ -1,55 +1,32 @@
 import pytest
 
 
-def test_sunrise_sunset():
+def test_sunrise_sunset_skyfield():
```

Comment:
I would leave the name as is. You are not testing anything specific to the implementation?, just the results of the glidertools function, so the name might induce confusion.